modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - windows
+  bazel:
+    - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
```

Comment:
Could you add the build flag `--process_headers_in_dependencies`? It's supported by Bazel 7.2 and gives some assurance for header-only libraries.

Reply:
Done.